components/org.wso2.carbon.identity.sso.saml/pom.xml

```diff
@@ -301,6 +301,11 @@
             <artifactId>org.apache.felix.scr.ds-annotations</artifactId>
             <scope>provided</scope>
         </dependency>
```

Comment:
Why do we need junit? We are using testng for unit testing.

Reply:
Addressed with https://github.com/wso2-extensions/identity-inbound-auth-saml/pull/323/commits/144cd02142d0987b9ec0e5a6c5fc617d1abb314a